seed/api/v2_1/urls.py

```diff
@@ -19,7 +18,6 @@
 router.register(r'taxlots', TaxLotViewSet, basename="taxlots")
 router.register(r'scenarios', ScenarioViewSet, basename="scenarios")
 router.register(r'tax_lot_properties', TaxLotPropertyViewSet, basename="tax_lot_properties")
-router.register(r'portfolio_manager', PortfolioManagerViewSet, basename="portfolio_manager")
```

Comment:
is it being used by the ui anywhere? i dont thing so, im just checking

Reply:
no, it isn't. We have a settings var to disable v2 API endpoints... we should eventually just get rid of all v2* API endpoints. Everyone should be using v3.